mypy/checker.py

```diff
@@ -3569,6 +3569,7 @@ def check_subtype(self, subtype: Type, supertype: Type, context: Context,
                 return False
             extra_info = []  # type: List[str]
             note_msg = ''
+            notes = []
```

Comment:
You might need to add an explicit type comment `List[str]` here, otherwise self-check may fail.